code/src/components/Footer.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import styled from "styled-components";
+
+const FooterWrapper = styled.div`
```

Comment:
What a great way to make a reusable component. Put this somewhere accessible so that you can reach it for future projects fast and easy. I will take this with me to my future projects.